src/driver/sqljs/SqljsConnectionOptions.ts

```diff
@@ -0,0 +1,32 @@
+import {BaseConnectionOptions} from "../../connection/BaseConnectionOptions";
+
+/**
+ * Sqlite-specific connection options.
+ */
+export interface SqljsConnectionOptions extends BaseConnectionOptions {
+
+    /**
+     * Database type.
+     */
+    readonly type: "sqljs";
+
+    /**
+     * The database definition and data to import
+     */
+    readonly database?: Uint8Array;
+
+    /**
+     * True if file or localStorage should be updated automatically
+     */
+    readonly autoSave?: boolean;
```

Comment:
Maybe its better to do `boolean|number` and if its a number, its an interval?

Reply:
What would be happen, when it's set to true? Then the interval would have to be a default value, right?

Reply:
I think we can make `autoSave` to `boolean|Function` instead of two parameters

Reply:
Or maybe even `save: boolean|string|Function` where string is a location, what do you think about it?